src/neptune/utils.py

```diff
@@ -46,26 +46,38 @@
 logger = get_logger()
 
 
-def stringify_unsupported(value: Any) -> Union[StringifyValue, Mapping]:
+def stringify_unsupported(
+    value: Any,
+    expand: bool = False,
+) -> Union[StringifyValue, Mapping]:
     """Helper function that converts unsupported values in a collection or dictionary to strings.
 
     Args:
         value (Any): A dictionary with values or a collection
+        expand (bool, optional): If True, the function will expand series to store each item as an
+        enumerated key-value pair. Otherwise, the entire series will be logged as a string. Defaults to False.
```

Comment:
```suggestion
        expand (bool, optional): If True, the function expands series to store each item as an
        enumerated key-value pair. Otherwise, the entire series is logged as a string. Defaults to False.
```